### ISSUE TITLE
Type of 'await' operand must either be a valid promise or must not contain a callable 'then' member.

### ISSUE THREAD

**Issue by alfdocimo** [cosmicjs/cosmic-sdk-js#18]
Hi there! 👋 

Massive kudos on the new sdk! 🙌  Been using cosmicjs for a while now, and with new dashboard beta, I had to give the new sdk a shot!

I'm having a bit of trouble setting things up, seems like you cannot `await` on queries? 🤔 

For instance:

```ts
async function main() {
  let posts = await cosmic.objects
    .find({
      type: 'posts',
    })
    .props(['title', 'slug', 'metadata'])
    .limit(10);

  return posts;
}
``` 
TS will complain with:
```
Type of 'await' operand must either be a valid promise or must not contain a callable 'then' member.(1320)
```

I've created a minimal stackblitz project to recreate this:

https://stackblitz.com/edit/node-goqgyn?file=src/index.ts

Again, kudos on the awesome work! 💯

**Comment by tonyspiro** [comment]
Thank you @alfdocimo! Apologies, this issue slipped past us. We'll review and let you know when we have an update. Thank you for the report 🙌🏻

**Comment by tonyspiro** [comment]
Closing this issue, as it's been addressed in #19 @alfdocimo to see this fix, upgrade to the latest version with `npm i @cosmicjs/sdk@latest`
